extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -405,6 +441,22 @@ export class AmpAnalytics extends AMP.BaseElement {
     }
     const typeConfig = this.predefinedConfig_[type] || {};
 
+    // transport.iframe is only allowed to be specified in typeConfig, not
+    // the others. Allowed when running locally for testing purposes.
+    [defaultConfig, inlineConfig, this.remoteConfig_].forEach(config => {
```

Comment:
Interesting. I believe we want extraData to remain configurable on a per-creative basis. But if that is indeed the case, I can find a place to move extraData to when #7096 is ready.